Documentation/build-dependencies.rst

```diff
@@ -0,0 +1,321 @@
+..
+  SPDX-FileCopyrightText: 2018-2024 SeisSol Group
+
+  SPDX-License-Identifier: BSD-3-Clause
+
+.. _build_dependencies:
+
+Installing Dependencies
+=======================
+
+For compiling SeisSol, you will need the following dependencies during build:
+
+- A C++17-capable compiler. The following work:
+
+  - GCC (>= 9.0; tested: 13.2)
+  - ICX (tested: 2024.2)
+  - Clang (tested: 18, 19)
+  - NVHPC (tested: 24.09; currently still slow!)
+  - Cray CE (however: no commthread support; needs ``SEISSOL_COMMTHREAD=0``)
+  - ICC 2021.9 is supported up to v1.2.0 only
```

Comment:
now works

Reply:
You mean, it works now again?

Reply:
yes (you fixed it with https://github.com/SeisSol/SeisSol/pull/1210)